k8s/charts/jd-bureau/templates/deployment.yaml

```diff
@@ -30,6 +30,9 @@ spec:
           image: "{{ .Values.image }}"
           imagePullPolicy: {{ .Values.imagePullPolicy }}
           volumeMounts:
+          - name: azurekvjd
+            mountPath: /app/server/config/environment
```

Comment:
this isn't right, check the path in the helm chart in chart-library,

I think it's /mnt/secrets/${keyvault-name}

Reply:
/mnt/secrets/{{ .Values.keyvault.name }}